keyboard-color-switcher.py

```diff
@@ -10,9 +10,6 @@
 from kcc_cli.keyboard_backlight import KeyboardBacklight
 from kcc_cli.enums import Position
 
-if os.geteuid() != 0:
-    exit("You need to have root privileges to run this.")
-
```

Comment:
Since we remove this we may want to add to the README.md that you need to be part of `plugdev` or whatever the group for those files in /sys/

Reply:
Well, that's one way, perhaps.

However, I'm going to see if I can get it to work with the Polkit policy.